apps/sendgrid/src/inngest/functions/users/sync-users.ts

```diff
@@ -0,0 +1,106 @@
+import type { User } from '@elba-security/sdk';
+import { eq } from 'drizzle-orm';
+import { NonRetriableError } from 'inngest';
+import { logger } from '@elba-security/logger';
+import { getUsers } from '@/connectors/sendgrid/users';
+import { db } from '@/database/client';
+import { organisationsTable } from '@/database/schema';
+import { inngest } from '@/inngest/client';
+import { type SendgridUser } from '@/connectors/sendgrid/users';
+import { decrypt } from '@/common/crypto';
+import { getElbaClient } from '@/connectors/elba/client';
+import { env } from '@/common/env';
+
+const formatElbaUser = (user: SendgridUser): User => ({
+  id: user.username,
+  displayName: user.username,
+  email: user.email,
+  role: user.is_admin ? 'owner' : 'user',
```

Comment:
```suggestion
  role: user.is_admin ? 'admin' : 'user',
```

In sendgrid, we invite user as admin, not owner. We could use `user.user_type` ('admin' | 'owner' | 'teammate') that is more precise.